freqtrade/exchange/common.py

```diff
@@ -81,14 +80,39 @@ async def wrapper(*args, **kwargs):
                 count -= 1
                 kwargs.update({'count': count})
                 if isinstance(ex, DDosProtection):
-                    backoff_delay = calculate_backoff(count + 1, API_RETRY_COUNT)
-                    logger.info(f"Applying DDosProtection backoff delay: {backoff_delay}")
-                    await asyncio.sleep(backoff_delay)
+                    if "kucoin" in str(ex) and "429000" in str(ex):
+                        logger.warning(f"Kucoin 429 error, keeping count the same.")
+                        count += 1
```

Comment:
I am sure this fixes the issue for yourself (at least partially) - however - it'll make the issue overall worse - as the rate-limit is a global one (3000r/s for all users) - so if everyone starts using this, you might not get a value at all.

It's also assuming that retrying immediately will work - but assuming the exchange has a generic problem (maybe because everyone tries this, and it's then no longer responding) you've blocked the bot in this loop to never get out from.
So i think a "overall" limit should still be applied to this (give up after X requests) to avoid blocking a bot completely.

Reply:
Done. Yeah, it's a dangerous path to take